src/org/loklak/api/search/GithubProfileScraper.java

```diff
@@ -19,35 +19,58 @@
 
 package org.loklak.api.search;
 
+import java.io.BufferedReader;
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Set;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;    
-import javax.servlet.http.HttpServletResponse;
+import org.apache.http.client.utils.URIBuilder;
 import org.json.JSONArray;
 import org.json.JSONObject;
 import org.json.JSONTokener;
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
 import org.jsoup.nodes.Element;
 import org.jsoup.select.Elements;
-import org.loklak.server.APIException;
-import org.loklak.server.APIHandler;
-import org.loklak.server.AbstractAPIHandler;
-import org.loklak.server.Authorization;
+import org.loklak.data.DAO;
+import org.loklak.harvester.BaseScraper;
+import org.loklak.harvester.Post;
 import org.loklak.server.BaseUserRole;
-import org.loklak.server.Query;
-import org.loklak.susi.SusiThought;
-import org.loklak.tools.storage.JSONObjectWithDefault;
 
-public class GithubProfileScraper extends AbstractAPIHandler implements APIHandler {
+public class GithubProfileScraper extends BaseScraper {
 
-    private static final long serialVersionUID = -4166800345379685201L;
+    private final long serialVersionUID = -4166800345379685202L;
     private static final String GITHUB_API_BASE = "https://api.github.com/users/";
+    public List<String> termsList = null;
+
+    public GithubProfileScraper() {
+        super();
+        this.baseUrl = "https://github.com/";
+        String scraperName = "Github";
```

Comment:
Why are you using this as a local varaible. This should be a class property right?